src/Apps/ViewingRoom/Routes/Works/Components/ViewingRoomCarousel.tsx

```diff
@@ -11,29 +20,77 @@ interface ViewingRoomCarouselProps {
 const ViewingRoomCarousel: React.FC<ViewingRoomCarouselProps> = ({
   artwork: { images },
 }) => {
+  const computeScrollPercent = selectedIndex =>
+    ((selectedIndex + 1) / images.length) * 100
+  const [scrollPercent, setScrollPercent] = useState(computeScrollPercent(0))
+  const update = flowRight(setScrollPercent, computeScrollPercent)
+  const showProgressBar = images.length > 1
+
   return (
     <Box width="100%">
-      <Flex position="relative" height={550} my={2}>
-        <Arrow direction="left" />
-
-        <Flex>
-          {images.map(({ internalID, imageHref }) => {
+      <Flex
+        height={550}
+        maxWidth={breakpoints.lg}
+        m="auto"
+        my={2}
+        position="relative"
+      >
+        <Carousel
+          options={{
+            cellAlign: "center",
+            draggable: showProgressBar,
+            freeScroll: true,
+            groupCells: 1,
+            pageDots: false,
+          }}
+          data={images}
+          height="550px"
```

Comment:
Weird having to specify the height? — We don't have proper placeholders in here?

Reply:
By placeholder you mean a placeholder component that shows an approximate pre-renderered layout that then fills in once the images / individual components load / stabalize?

Reply:
Oh, yeah, I was actually assuming this would have data in it already when the page loads for SSR, in which case you'd see properly sized image placeholders or whatever. Though if this is client-side only then the explicit height makes sense.

Reply:
It can be both, depending on where the user enters -- direct URL = SSR pass, but then navigating from elsewhere everything is on the client.